__test__/component/form/CreateCustomerForm.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { CreateCustomerForm } from '@/components/form/CustomerForm';
+import sinon, { spy } from 'sinon';
+import { Customer } from '@/lib/types/apiTypes';
+import { ApiUpdateResponse } from '@/lib/types/hooks';
+
+const customerEndpointURL = `${process.env.NEXT_PUBLIC_API_URL}/customer`;
+const bodySpy = sinon.spy((body) => body);
+const pathSpy = sinon.spy((path) => path);
```

Comment:
Here we define a couple of "spies". Spy is essentially just a function that does some extra bookkeeping. What we care about here, is that after `bodySpy()` it's possible to tell that `bodySpy` has been invoked by taking a look at `bodySpy.getCalls()`.